apps/itest/mix.exs

```diff
@@ -23,6 +23,7 @@ defmodule Itest.MixProject do
 
   defp deps() do
     [
+      # ex plasma based on 0.2.0 but with rust deps
       {:ex_plasma, git: "https://github.com/omisego/ex_plasma.git", ref: "5e94c4fc82dbf26cb457b30911505ec45ec534ea"},
```

Comment:
shouldn't you make ex_plasma also configurable? Because 0.2.0 only support uint 160 exit itds

Reply:
ex_plasma doesn't read any contract data